tests/models/test_hparams.py

```diff
@@ -536,9 +536,6 @@ def __init__(self, hparams):
 class SubClassVarArgs(SuperClassPositionalArgs):
     """Loading this model should accept hparams and init in the super class."""
 
-    def __init__(self, *args, **kwargs) -> None:
-        super().__init__(*args, **kwargs)
```

Comment:
this is here intentionally!

Reply:
can we have a comment that it is mandatory?